src/status_im/subs/chats.cljs

```diff
@@ -241,6 +241,13 @@
       :message-pin-enabled              message-pin-enabled
       :can-delete-message-for-everyone? can-delete-message-for-everyone?})))
 
+(re-frame/reg-sub
+ :chat/current-chat-one-to-one?
+ :<- [:chats/current-raw-chat]
+ (fn [current-chat]
+   (not (or (chat.events/group-chat? current-chat)
+            (chat.events/public-chat? current-chat)))))
```

Comment:
`(= (:chat-type current-chat) constants/one-to-one-chat-type))` can also be used